src/main.js

```diff
@@ -43,6 +43,12 @@ takeMeBackButton.addEventListener('click', function(){
 
 savePosterButton.addEventListener('click', savePoster);
 
+document.addEventListener('click', function(e){
+  if(e.target && e.target.parentElement.classList.contains("saved-poster")){
+    removePoster(e.target);
+  }
```

Comment:
This suggestion will have to be made with my next comment. If you rewrite this to pass `e.target.parentElement` (which would be the `article` element), you can...

Reply:
@Ryan-D-Miller in our `.poster-img` class (style.css line 51) - please remove `height: 450px` and `width: 800px`. Add in `max-width: 80%;`